alvr/graphics/src/lobby.rs

```diff
@@ -373,6 +404,7 @@ impl LobbyRenderer {
         view_params: [LobbyViewParams; 2],
         hand_data: [(Option<DeviceMotion>, Option<[Pose; 26]>); 2],
         body_skeleton_fb: Option<Vec<Option<Pose>>>,
+        body_skeleton_bd: Option<Vec<Option<Pose>>>,
```

Comment:
Since body_skeleton_fb and body_skeleton_bd have the same type, they could be the same parameter. to differentiate between pico and meta you could add a new TrackingType enum with `{ Meta, Pico }` variants, and pass this as `body_tracking_type: TrackingType`. Thanks to this parameter you can decide which skeleton mapping to use.